bats/fb-test-foreman-rex.bats

```diff
@@ -8,8 +8,8 @@ load foreman_helper
 
 @test "run 'uptime' via Remote Execution" {
   FOREMAN_VERSION=$(tForemanVersion)
-  if ! tIsVersionNewer "${FOREMAN_VERSION}" 3.3; then
-    job_template='Run Command - SSH Default'
+  if ! tIsVersionNewer "${FOREMAN_VERSION}" 3.7; then
+    skip "Skipping test for versions older than 3.8"
```

Comment:
Why skipping it here? We only care for versions newer than 3.3, so we can do `job_template='Run Command - Script Default'` unconditionally and drop the whole `if/else` block